src/build/arg/mod.rs

```diff
@@ -599,24 +599,24 @@ impl<'help> Arg<'help> {
     /// The above example displays
     ///
     /// ```text
-    /// helptest
+    /// prog
     ///
     /// USAGE:
-    ///    helptest [FLAGS]
+    ///     prog [FLAGS]
     ///
     /// FLAGS:
-    ///    --config
-    ///         The config file used by the myprog must be in JSON format
-    ///         with only valid keys and may not contain other nonsense
-    ///         that cannot be read by this program. Obviously I'm going on
-    ///         and on, so I'll stop now.
+    ///         --config
+    ///             The config file used by the myprog must be in JSON format
+    ///             with only valid keys and may not contain other nonsense
+    ///             that cannot be read by this program. Obviously I'm going on
+    ///             and on, so I'll stop now.
     ///
-    /// -h, --help
-    ///         Prints help information
-    ///
-    /// -V, --version
-    ///         Prints version information
+    ///     -h, --help
+    ///             Prints help information
```

Comment:
Why is there additional space in the start of the line now?

Reply:
Oh, looks like it is also there in other applications.

Reply:
The original doc is wrongly indented apparently.

Reply:
Yep, I just fixed it while I was here since it was my test case. The binary name was wrong too.